great_expectations/rule_based_profiler/domain_builder/categorical_column_domain_builder.py

```diff
@@ -166,7 +166,7 @@ def _generate_metric_configurations_to_check_cardinality(
         self,
         batch_ids: List[str],
         column_names: List[str],
-    ) -> List[Dict[str, List[MetricConfiguration]]]:
+    ) -> Dict[str, List[MetricConfiguration]]:
```

Comment:
Please also change the docstring to match the new return type

Reply:
@anthonyburdi Done (in next commit).  Thanks!